http_requests_base.py

```diff
@@ -97,10 +97,16 @@ def _make_request(self, signal):
                     r.status_code
                 )
             )
+            return self._process_response(r, signal)
```

Comment:
Before this would have always returned `None`. Now it's going to return the results of `_process_response`. Is this a desirable change? This PR should just be adding log messages but this seems to be changing functionality.